bin/upgrade-db

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+
+import shutil
+import sqlite3
+
+
+def main() -> None:
+    import argparse
+
+    parser = argparse.ArgumentParser(description="Upgrade an existing database\
+            file to version 3")
+    parser.add_argument("dbfiles", nargs="+", type=str,
+                        help="database file(s) to read")
+    parser.add_argument("--overwrite", action="store_true",
+                        help="""overwrite the passed in database file""")
+    parser.add_argument("--suffix", type=str, required=False,
+                        help="""a suffix to append to the filename of the
+                        newly upgraded database file""")
+
+    args = parser.parse_args()
+
+    if args.overwrite and args.suffix:
+        raise argparse.ArgumentError(
+                argument=None,
+                message="Cannot overwrite file and use a suffix"
+                )
+
+    if args.suffix == "":
+        raise NameError("Specify non-empty suffix, "
+                        "overwriting db files is performed with "
+                        "the flag '--overwrite'")
+
+    suffix = args.suffix or "_upgrade"
+
+    for dbfile in args.dbfiles:
+        new_db = upgrade_db(
+                dbfile, suffix=suffix, overwrite=args.overwrite
+                )
+
+        new_db.commit()
+
+
+def upgrade_db(
```

Comment:
Please move this function to `logpyle/upgrade_db.py`

Reply:
Moved